jerry-core/ecma/base/ecma-property-hashmap.c

```diff
@@ -539,8 +539,6 @@ ecma_property_hashmap_find (ecma_property_hashmap_t *hashmap_p, /**< hashmap */
       JERRY_ASSERT (entry_index != start_entry_index);
 #endif /* !JERRY_NDEBUG */
     }
-
-    JERRY_UNREACHABLE ();
```

Comment:
Why can this be removed?

Reply:
Because this is a while(true) loop with `return`s the only way out. There is no unreachable assert after the next while(true) loop either.